src/test/java/core/basesyntax/strategy/operations/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.strategy.operations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.Operation;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private static final Operation VALID_OPERATION = Operation.BALANCE;
+    private static final String VALID_FRUIT = "apple";
+    private static final int VALID_QUANTITY = 20;
```

Comment:
Consider add tests to check:
what if we put null fruit, zero or negative quantity?

Then, if we will have several tests working with Storage, to check them independently, we have to clear storage in `@AfterEach` method

Reply:
If I already checked "null fruit, zero or negative quantity" in FruitTransaction constructor, I can't to get invalid FruitTransaction object. So should I check these fields in each Handler's tests?

Reply:
Not necessarily, but you can make sure that there will be an exception here if the value is not valid.

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/121733725/ab1ee08a-98d0-4a8e-9b45-08749068dd3c)

I created this checks for each Handler class, but I had made alredy checking  in FruitTransaction constuctor, so I can't create invalid FruitTransaction objects. 
So, should I refuse from invalid cases in handlers, or in FruitTransaction's constructor?